baybe/strategies/base.py

```diff
@@ -62,4 +62,4 @@ def allow_recommending_already_measured(self):
 
 # Register (un-)structure hooks
 converter.register_unstructure_hook(Strategy, unstructure_base)
-converter.register_structure_hook(Strategy, structure_strategy)
+converter.register_structure_hook(Strategy, structure_recommender_protocol)
```

Comment:
is this necessary? Strategy already has a structure hook (via `RecommenderProtocol` inheritance) which is also the same function?